src/plugins/intel_gpu/tests/functional/behavior/infer_request.cpp

```diff
@@ -8,7 +8,6 @@
 #include "common_test_utils/node_builders/activation.hpp"
```

Comment:
You have updated  `src/plugins/intel_cpu/thirdparty/mlas`. Please revert

Reply:
thx. I missed it